src/commands/notify.yml

```diff
@@ -77,6 +77,13 @@ parameters:
     type: string
     default: Slack - Sending Notification
     description: Specify a custom step name for this command, if desired
+  thread_id:
+    type: string
+    default: ""
+    description: |
+      When set, the first `notify` with a given `thread_id` will appear as a regular slack message.
+      Any subsequent `notify` usage with the same identifier will be posted within the initial message's thread.
+      `thread_id` should be set to any arbitrary string to help you identify different threads. See examples for more information.
```

Comment:
@atanass could we add a quick note here to mention that enabling this parameter will use a small amount of caching?

Reply:
Sure thing.